### PR TITLE
NR-143043: Updated Nodejs dashboards and alerts

### PR DESCRIPTION
**Pull request by RamanaReddy8801**
# Summary

Ticket URL:https://issues.newrelic.com/browse/NR-143043
Description: This PR has the updated the Nodejs Dashboards and Alerts.

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/node-js/LowApdexScore.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: Apdex Score
 
 description: |+
-  This alert is triggered when the Apdex score is below 0.5 for 5 minutes
+  This alert is triggered when the Apdex score is down for at least 5 minutes.
 type: STATIC
 nrql:
   query: "SELECT apdex(duration, t: 0.5) FROM Transaction WHERE appName like '%'"
```
Comment (pkosanam) [inline alert-policies/node-js/LowApdexScore.yml:7]: should this be `WHERE appName` or `FACET appName` ?

File: quickstarts/node-js/node-js/config.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ description: |
   Our tool also provides other essential productivity and efficiency-enhancing innovations. Service Maps acquaints developers with their system's architecture providing vital insights to identify issues quickly. Error Analytics granularly pinpoints offending lines of code, relieving developers of a potentially arduous task so that they can concentrate on resolving issues. Our solution offers historical data that could prove essential in process improvement in addition to real-time information.
 summary: |
   New Relic's Node.js monitoring quickstart provides essential tools to monitor Node.js including multiple high-value alerts and informative dashboards to help developers visualize essential metrics and act on potential issues quickly.
-icon: logo.svg
+icon: logo.png
```
Comment (pkosanam) [inline quickstarts/node-js/node-js/config.yml:25]: any specific reason to change from svg to png ?

**Review by pkosanam — CHANGES_REQUESTED** [review]

File: quickstarts/node-js/node-js/config.yml (hunk 2 above)
Comment (RamanaReddy8801) [reply quickstarts/node-js/node-js/config.yml:25]: Yes, previous logo is broken.
Screenshot 2023-08-29 at 11 01 13 AM

File: alert-policies/node-js/LowApdexScore.yml (hunk 1 above)
Comment (RamanaReddy8801) [reply alert-policies/node-js/LowApdexScore.yml:7]: WHERE appName

File: alert-policies/node-js/HighCpuUtilization.yml [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
 
 description: |+
-  This alert is triggered when the CPU Utilization is above 90%.
+  This alert is triggered when the CPU utilization exceeds 80% for 5 minutes.
 
 type: STATIC
 nrql:
-  query: "SELECT rate(sum(apm.service.cpu.usertime.utilization), 1 second) * 100
-    as cpuUsage FROM Metric WHERE appName like '%'"
+  query: "SELECT latest(apm.service.cpu.usertime.utilization) * 100 as cpuUsage FROM Metric WHERE appName like '%'"
```
[the remark this answers is not in the document]
Comment (RamanaReddy8801) [reply alert-policies/node-js/HighCpuUtilization.yml:8]: Hi @pkosanam, The suggested query is not displaying any data

**Review by pkosanam — APPROVED** [review]
LGTM

**Review by mdumpati — APPROVED** [review]
LGTM

**Review by seemantk — APPROVED** [review]
lgtm

**Pull request merged**